sympy/solvers/inequalities.py

```diff
@@ -396,7 +396,17 @@ def solve_univariate_inequality(expr, gen, relational=True):
 
     """
 
-    from sympy.solvers.solvers import solve, denoms
+    from sympy.solvers.solvers import denoms
+    from sympy.solvers.solveset import solveset
+
+    def solve_replacement(f_x, x):
```

Comment:
The name `solve_replacement` makes sense in the context of this PR, but will it make sense a year from now in this file when people come across it?